### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by jedgarpark**
This is a demo of basic MIDI functionality on Macropad using `adafruit_macropad` library for keys, LEDs, knob, knob switch, display, and MIDI. Sends noteon/noteoff, program change, cc, pitchbend.

### CODE REVIEW

**Review by kattni — CHANGES_REQUESTED** [review]
This is lovely! I think the thing that should determine its home, though, is whether or not it's destined for a guide. If so, let's merge it here. If not, I think it is better suited to the MacroPad library examples. Let me know!

**Comment by jedgarpark** [comment]
@kattni cool -- how about we put it in the main Macropad guide then. I can write up a page for it.

**Review by kattni — APPROVED** [review]
Ooh, that would be brilliant! Please put the page under "CircuitPython Essentials" titled "MacroPad MIDI". We'll title the examples that use the MacroPad library similarly to identify them.

File: Macropad_MIDI_Tester/code.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# SPDX-FileCopyrightText: 2021 John Park for Adafruit Industries
+# SPDX-License-Identifier: MIT
+# Macropad MIDI Tester
+# Play MIDI notes with keys
+# Click encoder to switch modes
+# Turn encoder to adjust CC, ProgramChange, or PitchBend
+from adafruit_macropad import MacroPad
+from rainbow import colorwheel
```
Comment (kattni) [inline Macropad_MIDI_Tester/code.py:8]: This has been updated to `rainbowio`. Please update it here. Thanks!

**Review by kattni — CHANGES_REQUESTED** [review]
Woops! One thing below.

File: Macropad_MIDI_Tester/code.py (hunk 1 above)
Comment (jedgarpark) [reply Macropad_MIDI_Tester/code.py]: on it

**Review by kattni — APPROVED** [review]
Thanks!

**Pull request merged**